integration-cli/docker_cli_run_test.go

```diff
@@ -493,6 +493,21 @@ func TestVolumesFromGetsProperMode(t *testing.T) {
 	logDone("run - volumes from ignores `rw` if inherrited volume is `ro`")
 }
 
+// Test for GH#10618
+func TestRunNoDupVolumes(t *testing.T) {
+	cmd := exec.Command(dockerBinary, "run", "-v", "/etc:/someplace", "-v", "/usr/lib:/someplace", "busybox", "echo", "hi")
```

Comment:
@estesp can we bindmount two folders we create please? I don't like to bindmount /etc even for tests :)